tasks/create-cert-webroot.yml

```
@@ -0,0 +1,30 @@
+---
+- name: Check if certificate already exists.
+  stat:
+    path: /etc/letsencrypt/live/{{ cert_item.domains | first }}/cert.pem
+  register: letsencrypt_cert
+
+- name: Create webroot directory if it doesn't exist yet
+  file:
+    path: "{{ cert_item.webroot | default(certbot_webroot) }}"
```

```suggestion
    path: "{{ _certbot_cert_item.webroot | default(certbot_webroot) }}"
```

Private variables should be prefixed with `_certbot_`